app/models/uphold_connection.rb

```diff
@@ -95,6 +95,7 @@ def disconnect_uphold
       default_currency_confirmed_at: nil,
       default_currency: nil,
     )
+    publisher.update(wallet_provider: nil)
```

Comment:
I think the `wallet_provider` should only be cleared if the uphold_connection is the selected wallet_provider

Reply:
https://github.com/brave-intl/publishers/pull/2857/commits/5243340640e464a6d87ed6a8aec1277e870721a5